electro

```diff
@@ -1,19 +1,19 @@
-#!/usr/bin/env bash
+#!/bin/bash
 set -euo pipefail
 IFS=$'\n\t'
 
 # configuration variables, read from config file
 ELECTRO_INSTANCE_ID="i-xxx"
 HOST="prima"
-SSH_HOST="dev-prima"
-# opzionali
+PING_VPN="10.254.0.250"
```

Comment:
togliendo SSH_HOST hai uniformato l'host definito nella config di SSH e l'host che serve per usare in locale e far puntare alla macchina remota. se è quello che si voleva ok